web-client/src/presenter/computeds/formattedWorkQueue.test.js

```diff
@@ -0,0 +1,864 @@
+    it('should return /complete if document is in progress and user is docketclerk', () => {
+      const { permissions } = getBaseState(docketClerkUser);
+
+      const result = getWorkItemDocumentLink({
+        applicationContext,
+        permissions,
+        workItem: {
+          ...baseWorkItem,
+          document: {
+            ...baseDocument,
+            category: 'Miscellaneous',
+            documentTitle: 'Administrative Record',
+            documentType: 'Administrative Record',
+            eventCode: 'ADMR',
+            isFileAttached: false,
+            isPaper: true,
+            pending: false,
+            receivedAt: '2018-01-01',
+            relationship: 'primaryDocument',
+            scenario: 'Standard',
+          },
+          isInitializeCase: false,
+          isQC: true,
+          messages: [baseMessage],
+          section: 'docket',
+        },
+        workQueueToDisplay: {
+          box: 'inProgress',
+          queue: 'section',
+          workQueueIsInternal: false,
+        },
+      });
+      expect(result).toEqual('/complete');
+    });
+
+    it('should return default edit link if document is in progress and user is petitionsClerk', () => {
+      const { permissions } = getBaseState(petitionsClerkUser);
+
+      const result = getWorkItemDocumentLink({
+        applicationContext,
+        permissions,
+        workItem: {
+          ...baseWorkItem,
+          document: {
+            ...baseDocument,
+            category: 'Miscellaneous',
+            documentTitle: 'Administrative Record',
+            documentType: 'Administrative Record',
+            eventCode: 'ADMR',
+            isFileAttached: false,
+            isPaper: true,
+            pending: false,
+            receivedAt: '2018-01-01',
+            relationship: 'primaryDocument',
+            scenario: 'Standard',
+          },
+          isInitializeCase: false,
+          isQC: true,
+          messages: [baseMessage],
+          section: 'docket',
+        },
+        workQueueToDisplay: {
+          box: 'inProgress',
+          queue: 'section',
+          workQueueIsInternal: false,
+        },
+      });
+      expect(result).toEqual('');
+    });
+
+    it("should return /edit if document is an external doc that has not been qc'd and user is docketclerk", () => {
+      const { permissions } = getBaseState(docketClerkUser);
+
+      const result = getWorkItemDocumentLink({
+        applicationContext,
+        permissions,
+        workItem: {
+          ...baseWorkItem,
+          document: {
+            ...baseDocument,
+            category: 'Miscellaneous',
+            documentTitle: 'Administrative Record',
+            documentType: 'Administrative Record',
+            eventCode: 'ADMR',
+            pending: false,
+            receivedAt: '2018-01-01',
+            relationship: 'primaryDocument',
+            scenario: 'Standard',
+          },
+          isInitializeCase: false,
+          isQC: true,
```

Comment:
This is confusing, I agree, but I'm not sure how to make it less confusing. `isQC: true` means this work item is a QC work item that goes in a document queue rather than a message work item that goes in a message queue. A QC work item has been QC'd when the message attached the work item is marked as read and completed.

Reply:
Yeah, isQC essentially describes the state of the workItem - it is a QC workItem (shows in the QC box).

Reply:
Hmm... What about this?

`...has not been qc'd (isQC is true) and user is...`

Is that too horrible?

My test description aesthetic is currently at odds with my desire to have easily understood tests.